.github/workflows/frontend-deploy.yml

```
@@ -0,0 +1,32 @@
+name: Frontend - Deploy ecrc-api
```

```suggestion
name: Frontend - Deploy ecrc-frontend
```
Small copy paste error